config/locales/forms/en.yml

```diff
@@ -137,7 +136,7 @@ en:
       title: Confirm your account
       welcome_back: Welcome back
       welcome_back_description: '<p>If you have received your letter, please enter
-        your one-time code below. </p>  <p>If your letter hasn’t arrived yet,
+        your one-time code below. </p><p>If your letter hasn’t arrived yet,
```

Comment:
There were some irregular line endings in here that VS Code would complain about each time I opened the file. Doesn't seem like they were intentional or required.

![image](https://user-images.githubusercontent.com/1779930/207073210-f8d6eef8-59e8-4e11-8416-3a03875408c0.png)

![image](https://user-images.githubusercontent.com/1779930/207073270-c28f778a-97dd-4431-8e03-865ec5ea1bb6.png)